hw_7/task_1.js

```diff
@@ -0,0 +1,46 @@
+function separateLogs() {
+    console.log("================================");
+  }
+  
+ // Task 1 
+function mergedArrays(...array) {
+    const newArray = [];
+    for (const num of array) {
+       newArray.push(...num);
+    }
+return newArray;
+}
+console.log (mergedArrays([1,2], [3,4], [5,6]));
+
+separateLogs();
+
+//Task 2
+function changeRegister(sentence) {
+    const words = sentence.split(' ');
+    const result = [];
+
+    for (let i = 0; i < words.length; i++) {
+        if (i === 0) {
+            result.push(words[i].toLowerCase()); //как я поняла, тут мы первое слово приводим к нижнему регистру
+        } else {
+            result.push(words[i].charAt(0).toUpperCase() + words[i].slice(1)); // тут остальные слова должны начинаться с большой буквы
```

Comment:
Отбрасьіваем такой случай, что в словах будут заглавньіе буквьі `enGinEer`
```suggestion
            result.push(words[i].charAt(0).toUpperCase() + words[i].slice(1).toLowerCase()); // тут остальные слова должны начинаться с большой буквы
```